tests/unittests/test_merging.py

```diff
@@ -100,7 +100,7 @@ def make_dict(max_depth, seed=None):
     return _make_dict(0, max_depth, rand)
 
 
-class TestSimpleRun(helpers.ResourceUsingTestCase):
+class TestSimpleRun(helpers.CiTestCase):
```

Comment:
Can we also drop the CiTestCase subclass here or is this refactor too challenging for this PR?

Reply:
Done